content/blog/rust-conferences-2025/index.md

```diff
@@ -0,0 +1,258 @@
+- **Status**: To be announced
+- **Links**: [Website](https://rustfest.ch/) 
+
+---
+
+So there you have it, our complete guide to all Rust developer conferences, workshops, and community meetups happening around the world in 2025. Whether you're interested in blockchain, embedded systems, or web development with Rust, there's an event for you.
+Let us know if we missed any events or if you have any updates to share. We'll see you at the next Rust conference!
+
+*Note: This list will be updated as more information becomes available. Some conferences are yet to announce their exact dates, venues, or ticket prices.*
+
+<script>
+// Add days until conference start
+function addDaysUntilConference() {
+    // Get all elements containing conference dates (assumes they're in <strong> tags after "When")
+    const dateElements = document.querySelectorAll('li strong:contains("When")');
+    
+    dateElements.forEach(element => {
+        // Get the date text that follows the "When: " label
+        const dateText = element.nextSibling.textContent.trim();
+        
+        // Extract the start date (handles both single dates and ranges)
+        const startDate = dateText.split('-')[0].trim().split(',')[0];
+        
+        // Parse the date
+        const confDate = new Date(startDate + ", 2025");
+        const today = new Date();
+        
+        // Calculate days difference
+        const diffTime = confDate - today;
+        const diffDays = Math.ceil(diffTime / (1000 * 60 * 60 * 24));
+        
+        // Add the days count in parentheses
+        if (diffDays > 0) {
+            element.nextSibling.textContent += ` (in ${diffDays} days)`;
+        }
+    });
+}
+
+// Alternative simpler version without jQuery dependency
+function addDaysUntilConference() {
+    // Get all text nodes containing dates
+    const walker = document.createTreeWalker(
+        document.body,
+        NodeFilter.SHOW_TEXT,
+        null,
+        false
+    );
+
+    let node;
+    const datePattern = /(?:January|February|March|April|May|June|July|August|September|October|November|December)\s+\d{1,2}(?:-\d{1,2})?,\s+2025/i;
+
+    while (node = walker.nextNode()) {
+        if (datePattern.test(node.textContent)) {
+            const dateText = node.textContent.match(datePattern)[0];
+            const startDate = dateText.split('-')[0].split(',')[0];
+            
+            const confDate = new Date(startDate + ", 2025");
+            const today = new Date();
+            
+            const diffTime = confDate - today;
+            const diffDays = Math.ceil(diffTime / (1000 * 60 * 60 * 24));
+            
+            if (diffDays > 0) {
+                node.textContent = node.textContent.replace(
+                    dateText,
+                    `${dateText} (in ${diffDays} days)`
+                );
+            }
+        }
+    }
+}
+
+// Run when the document is loaded
+document.addEventListener('DOMContentLoaded', addDaysUntilConference);
```

Comment:
You can also just call the function now. At the point where this script is parsed, the DOM is already present, and all the elements you want to modify have been loaded. Then there _really_ is no blink.

Reply:
done